Sources/Core/Extensions/UserDefaults+Extensions.swift

```diff
@@ -23,4 +23,5 @@ extension UserDefaults: DataPersisting {}
 protocol DataPersisting {
```

Comment:
I think UserDefaults has those functions already.

Reply:
Correct. This is just a lightweight wrapper so we can spy UserDefaults in a test. There’s an additional test to make sure we actually use UserDefaults in ‘production’

Reply:
If this is only for testing, I'd suggest to use mock instead.